src/components/Tooltip/Tooltip.tsx

```diff
@@ -193,8 +193,8 @@ const Tooltip = ({
     position: "absolute",
     // Initially position the tooltip of the screen in case it gets shown
     // before setting the position.
-    left: -99999999999999,
-    top: -99999999999999,
+    left: -9999999,
```

Comment:
Not sure why, this alongside https://github.com/canonical/vanilla-framework/pull/4639 introduced in 3.11.1 was causing cypress to show false positives when checking visibility of tooltip.

Reducing the left offset seems to fix that for now…